databases/sql-1.md

```diff
@@ -55,21 +55,21 @@ Most software applications store data of some sort -- user data, content, etc. D
 
 1. Work through the [Codecademy SQL Tutorial](https://www.codecademy.com/learn/learn-sql) **Section 1 (Manipulation)**:
 
-1. PostgreSQL is a popular, open-source version of SQL. Try it out:
+2. PostgreSQL is a popular, open-source version of SQL. Try it out:
```

Comment:
It's fine to leave in the re-numbered list items, but just fyi, markdown automatically puts in the right list numbers for you when rendering the page. People often use all 1's in markdown lists to make it easier to add intermediate steps later without needing to renumber the list.

Reply:
@alodahl  I believe this issue is resolved and ready for merge